resources/startstopserver.ts

```diff
@@ -20,63 +20,64 @@ import { ECSClient, UpdateServiceCommand } from '@aws-sdk/client-ecs';
 import { APIGatewayEvent, APIGatewayProxyHandler, Context } from 'aws-lambda';
 
 //Set the AWS Region
-const REGION = process.env.REGION; 
+const REGION = process.env.REGION;
 
-const SERVICE_NAME = process.env.SERVICE_NAME;  
-const CLUSTER_ARN = process.env.CLUSTER_ARN; 
+const SERVICE_NAME = process.env.SERVICE_NAME;
+const CLUSTER_ARN = process.env.CLUSTER_ARN;
 const PASSWORD = process.env.PASSWORD;
 
-const handler: APIGatewayProxyHandler = async (event: APIGatewayEvent, context: Context) => {
-    console.log("request: " + JSON.stringify(event));
-    let responseCode = 400;
-    let message = "authentication failed";
+export const handler: APIGatewayProxyHandler = async (event: APIGatewayEvent, context: Context) => {
+  console.log("request: " + JSON.stringify(event));
+  let responseCode = 400;
+  let message = "authentication failed";
 
-    const params = {
-        desiredCount: 1,
-        service: SERVICE_NAME,
-        cluster: CLUSTER_ARN
-    }
-    
-    if (event.queryStringParameters && event.queryStringParameters.desiredCount !== undefined) {
-        let count = Math.min(Math.max(+event.queryStringParameters.desiredCount, 0), 1);
-        params.desiredCount = count;
-        console.log("changing desiredCount to " + count);
-    }
+  const params = {
+    desiredCount: 1,
+    service: SERVICE_NAME,
+    cluster: CLUSTER_ARN
+  }
 
-    if (event.queryStringParameters && event.queryStringParameters.key) {
-        let key = event.queryStringParameters.key;
-        if (key == PASSWORD) {
-            const client = new ECSClient({ region: REGION });
-            console.log("starting service " + JSON.stringify(params));
-            message = "authentication success";
-            responseCode = 200;
+  if (event.queryStringParameters && event.queryStringParameters.desiredCount !== undefined) {
+    let count = Math.min(Math.max(+event.queryStringParameters.desiredCount, 0), 1);
+    params.desiredCount = count;
+    console.log("changing desiredCount to " + count);
+  }
 
+  if (event.queryStringParameters && event.queryStringParameters.key) {
+    let key = event.queryStringParameters.key;
+    if (key == PASSWORD) {
```

Comment:
Just realized, this should be `===` 

Ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness